arcade/gui/surface.py

```diff
@@ -83,7 +83,7 @@ def width(self) -> int:
     def height(self) -> int:
         return self._size[1]
 
-    def clear(self, color: RGBA255 = TRANSPARENT_BLACK):
+    def clear(self, color: RGBOrA255 = TRANSPARENT_BLACK):
```

Comment:
This was left as `RGBA255` intentionally as a nudge toward full RGBA. I don't have the benchmarks handy, but I believe it may be slightly faster to use RGBA colors in some cases. The `RGBAOrA255` alias is only in the defined for completeness reasons in case of edge cases that demand it.

Reply:
How about we extract that to an issue and tackle in a different PR? That way, we can keep this PR focused solely on enabling pyright while preserving existing behavior.

Reply:
Nevermind me, the change to use `RGBA255` was tiny, I made the change.